pkg/api/v1/helper/helpers.go

```diff
@@ -513,3 +519,70 @@ func StorageNodeAffinityToAlphaAnnotation(annotations map[string]string, affinit
 	annotations[v1.AlphaStorageNodeAffinityAnnotation] = string(json)
 	return nil
 }
+
+// ReorganizeTaints returns the updated set of taints, taking into account old taints that were not updated,
+// old taints that were updated, old taints that were deleted, and new taints.
+func ReorganizeTaints(node *v1.Node, overwrite bool, taintsToAdd []v1.Taint, taintsToRemove []v1.Taint) (string, []v1.Taint, error) {
```

Comment:
@ravisantoshgudimetla Can you please show more detail for what is the intention of putting those functions such as `ReorganizeTaints`, `addTaints`, `deleteTaints` and `ValidateNoTaintOverwrites` to the `helper` folder but not the `taintutil` like `ParseTaints`?

Is there are policy for this such as when to update `helper` and when to update `util`?